distributed/variable.py

```diff
@@ -191,6 +198,15 @@ def set(self, value, **kwargs):
         """
         return self.client.sync(self._set, value, **kwargs)
 
+    async def _is_set(self):
+        return await self.client.scheduler.variable_is_set(
+            name=self.name, client=self.client.id
+        )
+
+    def is_set(self, **kwargs):
```

Comment:
@fjetter Thanks for the review.

It's a good question. I'm not sure what that pattern is about. I noticed that the `get` and `set` variants do that pattern, and would cause the same error.

But yeah, also not too sure, I just went with following the established pattern. Happy to remove. Let me know.

Reply:
Yes, please remove it. For `get` we can at least pass timeout through but a cleaner way would be to be explicit. For set there is no excuse and we should remove it (not in this PR)

Reply:
Ok, seems strange. Maybe a byproduct from a past implementation. Anyway, I've pushed with the `kwargs` removed from the `is_set` signature. Thanks